src/monattack.cpp

```diff
@@ -4345,7 +4345,7 @@ bool mattack::riotbot(monster *z)
                 foe->i_add(handcuffs);
             } else {
                 handcuffs.item_tags.insert("NO_UNWIELD");
-                foe->wield(&(foe->i_add(handcuffs)));
+                foe->wield( foe->i_add( handcuffs ) );
```

Comment:
How does this part work with atomic `wield`? I assume on failure, the cuffs will end up in the inventory.
Couldn't it be simplified to just `foe->wield( handcuffs )`?

Later on it probably should handle failure to cuff, but that's not really relevant to this PR.